.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# @intlify/eslint-plugin-svelte Contributing Guide
+
+- [Issue Reporting Guidelines](#issue-reporting-guidelines)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [Development Setup](#development-setup)
+
+## Issue Reporting Guidelines
+
+- The issue list of this repo is **exclusively** for bug reports and feature requests. Non-conforming issues will be closed immediately.
+
+  - For simple beginner questions, you can get quick answers from [`#vue-i18n` channel of Discord](https://chat.vuejs.org/)
```

Comment:
I don't know where the svelte community is, and don't go there much. 😅 
So maybe this is a good place to enable GitHub Discussions on `eslint-plguin-svelte` and change the link to that.